docs/Parameters.rst

```diff
@@ -382,6 +386,8 @@ Learning Control Parameters
 
 -  ``min_gain_to_split`` :raw-html:`<a id="min_gain_to_split" title="Permalink to this parameter" href="#min_gain_to_split">&#x1F517;&#xFE0E;</a>`, default = ``0.0``, type = double, aliases: ``min_split_gain``, constraints: ``min_gain_to_split >= 0.0``
 
+   -  can be used to speed up training
+
    -  the minimal gain to perform split
```

Comment:
Swap these points please as we use `can be used ...` at the very end of param description.

Reply:
done!